tensorrt_llm/serve/openai_server.py

```
@@ -975,6 +995,42 @@ async def create_stream_response(generator, request: ResponsesRequest, sampling_
 
         return JSONResponse(content={"detail": "None"})
 
+    async def release_memory(self, request: MemoryUpdateRequest) -> JSONResponse:
```

Ruff (0.14.5)

Unused method argument: `request`

(ARG002)